lib/utils.pm

```diff
@@ -808,6 +809,22 @@ sub handle_login {
     send_key "ret";
 }
 
+sub handle_logout {
+    # hide mouse for clean logout needles
+    mouse_hide();
+    # logout
+    if (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'lxde')) {
+        my $command = check_var('DESKTOP', 'gnome') ? 'gnome-session-quit' : 'lxsession-logout';
+        x11_start_program("$command");    # opens logout dialog
+        assert_screen 'logoutdialog' unless check_var('DESKTOP', 'gnome');
+    }
+    else {
+        my $key = check_var('DESKTOP', 'xfce') ? 'alt-f4' : 'ctrl-alt-delete';
+        send_key_until_needlematch 'logoutdialog', "$key";    # opens logout dialog
```

Comment:
not sure if it is good idea to say "if you get to some unknown DE which we don't know how to handle just press ctrl+alt+delete" , also I am curious what combinations actually possible here ? We already hadnle gnome , lxde , xfce up to this point , so what else left ? only kde came to my mind , so what if we will clearly state here all types of DE's that we KNOW that we can handle and die in case something new ?